### PR TITLE
Add Hero section 

### PR DESCRIPTION
**Pull request by EthanMoveo**

### CODE REVIEW

File: src/components/Hero.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import styled, {css} from 'styled-components';
+
+import { colors } from './../constants/colors'; 
+import { fonts } from './../constants/fonts'; 
+import backgroundImageMobile from './../assets/pictures/Hero.png'
+import backgroundImageDesktop from './../assets/pictures/Hero-Desktop.png'
+
+const backgroundColorContainer = css`
+  background-color: #FFFFFFE0;
```
Comment (noyflaysher) [inline src/components/Hero.styles.tsx]: use variable

File: src/components/Hero.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+  @media (min-width: 1440px) {
+    background-image: url(${backgroundImageDesktop});
+    padding: 236px 332px;
+  }
+`;
+
+
+export const TextContainer = styled.div`
+  ${backgroundColorContainer}
```
Comment (noyflaysher) [inline src/components/Hero.styles.tsx]: use directly

File: src/components/Hero.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+`;
+
+export const SearchInputContainer = styled.div`
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  padding: 6px 12px;
+  gap: 8px;
+  width: 315px;
+  border: 0.5px solid #000000;
```
Comment (noyflaysher) [inline src/components/Hero.styles.tsx]: use variable for the color

File: src/components/Hero.styles.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  @media (min-width: 1440px) {
+    ${fonts.DESKTOP.H1}
+    font-size: 24px;
+    padding: 5px 0px;
+
+  }
+
+  &::placeholder {
+    ${fonts.MOBILE.H1}
+    font-size: 12px;
```
Comment (noyflaysher) [inline src/components/Hero.styles.tsx]: do you have another font with 12 px?

File: src/components/atoms/Header.styles.tsx [hunk 5]
```diff
@@ -9,6 +9,8 @@ export const HeaderContainer = styled.header`
   top: 0;
   height: 46px;
   padding: 11px 20px;
+  background-color: white;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: use color variable

Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Done

File: src/components/Hero.styles.tsx (hunk 4 above)
Comment (EthanMoveo) [reply src/components/Hero.styles.tsx]: No

File: src/components/Hero.styles.tsx (hunk 3 above)
Comment (EthanMoveo) [reply src/components/Hero.styles.tsx]: Done

File: src/components/Hero.styles.tsx (hunk 2 above)
Comment (EthanMoveo) [reply src/components/Hero.styles.tsx]: DONE

File: src/components/Hero.styles.tsx (hunk 1 above)
Comment (EthanMoveo) [reply src/components/Hero.styles.tsx]: DONE

File: src/components/Hero.styles.tsx [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+import { COLORS } from './../constants/colors'; 
+import { FONTS } from './../constants/fonts'; 
+import backgroundImageMobile from './../assets/pictures/Hero.png'
+import backgroundImageDesktop from './../assets/pictures/Hero-Desktop.png'
+
+
+export const HeroContainer = styled.section`
+  background-image: url(${backgroundImageMobile});
+  background-size: cover;
+  background-position: center;
```
Comment (noyflaysher) [inline src/components/Hero.styles.tsx]: do you need it ? check

File: src/components/atoms/Header.styles.tsx [hunk 7]
```diff
@@ -9,6 +10,8 @@ export const HeaderContainer = styled.header`
   top: 0;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:10]: why ?

File: src/components/Hero.styles.tsx (hunk 6 above)
Comment (EthanMoveo) [reply src/components/Hero.styles.tsx]: I removed it

File: src/components/atoms/Header.styles.tsx (hunk 7 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx:10]: Without top: 0, the sticky property does not fix the header at the top of the viewport

**Review by noyflaysher — APPROVED** [review]

**Pull request merged**